pyfr/plugins/base.py

```diff
@@ -62,6 +70,19 @@ def serialise(self, intg):
         return {}
 
 
+class BaseCLIPlugin:
+    name = None
+    systems = None
```

Comment:
Not sure this or the below are needed.

Reply:
They aren't needed at the present but in the future they probably will. But I can remove.

Reply:
I'm not quite sure how they can ever be used; in regular plugins they work as the integrator has a system which has a name, and this can be used to check compatibility.  However, for a plugin which is free to pick its own set of command line arguments there really is no analogue.  Indeed, it may not always be the case that the thing a plugin is working on is a solution!